.hound.yml

```diff
@@ -0,0 +1,7 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+  ignore_file: .eslintignore
+
+scss:
+  enabled: false
```

Comment:
@coolbeatz71 Why is Sass disabled?

Reply:
Ohh yes! @gracelungu It was the default configuration. I need to change it

Reply:
> Ohh yes! @gracelungu It was the default configuration. I need to change it

Fixed here https://github.com/andela/ah-kgl-wakanda-frontend/pull/5/commits/44f344ad685d3e58cc1ed704aaf2b66d01a2d41e